samples/app-hello-world/nodejs/appManifest/manifest.json

```diff
@@ -1,80 +1,119 @@
 {
-  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.16",
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/vDevPreview/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
   "version": "1.0.0",
   "id": "${{TEAMS_APP_ID}}",
   "packageName": "com.contoso.helloworld",
   "developer": {
-    "name": "Contoso",
-    "websiteUrl": "https://www.microsoft.com",
-    "privacyUrl": "https://www.microsoft.com/privacy",
-    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+      "name": "Contoso",
+      "websiteUrl": "https://www.microsoft.com",
+      "privacyUrl": "https://www.microsoft.com/privacy",
+      "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
   },
   "name": {
-    "short": "Hello World",
-    "full": "Hello World App for Microsoft Teams"
+      "short": "ElementRelationshipSet",
+      "full": "Hello World App for Microsoft Teams"
   },
   "description": {
-    "short": "Hello World App for Microsoft Teams",
-    "full": "This sample app provides a very simple app for Microsoft Teams. You can extend this to add more content and capabilities."
+      "short": "Hello World App for Microsoft Teams",
+      "full": "This sample app provides a very simple app for Microsoft Teams. You can extend this to add more content and capabilities."
   },
   "icons": {
-     "outline": "outline.png",
-    "color": "color.png"
+      "outline": "outline.png",
+      "color": "color.png"
   },
   "accentColor": "#60A18E",
-  "staticTabs": [
-    {
-      "entityId": "com.contoso.helloworld.hellotab",
-      "name": "Hello Tab",
-      "contentUrl": "https://${{BOT_DOMAIN}}/hello",
-      "scopes": [
-        "personal"
+  "elementRelationshipSet": {
+      "oneWayDependencies": [
```

Comment:
Let's add one more oneWayDepend scenario that's about a tab dependsOn ME (at ME level, instead of command level)